### PR TITLE
closes #172 Added unit test for snapshot iterator

### LINKED ISSUE #172: Add unit test for SnapshotIterator
This iterator needs a comprehensive unit test. The unit test does not have to use minifluo, it could just use the Accumulo SortedMapIterator.

### PR DESCRIPTION
**Pull request by keith-turner**

### CODE REVIEW

File: modules/accumulo/src/main/java/io/fluo/accumulo/iterators/SnapshotIterator.java [hunk 1]
```diff
@@ -32,12 +32,9 @@
 import org.apache.accumulo.core.iterators.IteratorEnvironment;
 import org.apache.accumulo.core.iterators.SortedKeyValueIterator;
 
-/**
- * 
- */
 public class SnapshotIterator implements SortedKeyValueIterator<Key,Value> {
   
-  private static final String TIMESTAMP_OPT = "timestampOpt";
+  static final String TIMESTAMP_OPT = "timestampOpt";
```
Comment (mikewalch) [inline modules/accumulo/src/main/java/io/fluo/accumulo/iterators/SnapshotIterator.java]: How about adding a `@VisibileForTesting` annotation here?

Comment (keith-turner) [inline modules/accumulo/src/main/java/io/fluo/accumulo/iterators/SnapshotIterator.java]: added in 1aa58b8

File: modules/accumulo/src/test/java/io/fluo/accumulo/iterators/SnapshotIteratorTest.java [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+      Assert.assertEquals(0, output.data.size());
+
+      output = new TestData(newSI(input, 17), range);
+      TestData expected = new TestData();
+      expected.add("0 f q1 DATA 11","i", range);
+      Assert.assertEquals(expected, output);
+
+      output = new TestData(newSI(input, 19), range);
+      expected = new TestData();
+      expected.add("0 f q1 DATA 11","i", range);
```
Comment (mikewalch) [inline modules/accumulo/src/test/java/io/fluo/accumulo/iterators/SnapshotIteratorTest.java]: How about changing this method name to `addIfInRange()`

**Comment by mikewalch** [comment]
+1

**Pull request merged**